operator/pkg/util/merge_iop.go

```diff
@@ -183,7 +183,6 @@ type (
 
 type proxyConfig struct {
 	DrainDuration                  *durationpb.Duration                    `json:"drainDuration" patchStrategy:"replace"`
-	ParentShutdownDuration         *durationpb.Duration                    `json:"parentShutdownDuration" patchStrategy:"replace"`
```

Comment:
I think we migght still need this in case user sets it

Reply:
Will remove it from proto as well

Reply:
Should I clean that up first or any issues with cleaning up?